docs/realm-configuration.md

```diff
@@ -60,7 +60,7 @@ Realm ui mappings is a map containing two list of ui mapping : a uiUserMapping l
 
 A UI mapping is defined as follow :
 
-name;helpTextTitle;helpText;path;type;modifiable;tag[;order;option:value...]
+name;helpTextTitle;helpText;path;type;modifiable;tag[;order;required;option:value...]
```

Comment:
while you're at it can you just update this doc for option=value instead of option:value ?